skyportal/models.py

```diff
@@ -2777,6 +2777,20 @@ def from_ascii(
 
 # should be accessible only by spectrumowner ^^
 
+SpectrumReducer.external_reducer = sa.Column(
+    sa.String,
+    nullable=True,
+    doc="The actual reducer, provided as free text, for the spectrum, separate "
+    "from the point-of-contact Fritz user designated as reducer",
```

Comment:
Maybe add something like "if the reducer is not a user in the database" for clarity?
Below as well